frontend/public/components/overview/resource-overview-page.tsx

```diff
@@ -9,19 +9,38 @@ import { NetworkingOverview } from './networking-overview';
 import { PodsOverview } from './pods-overview';
 import { resourceOverviewPages } from './resource-overview-pages';
 import { OverviewItem } from '@console/shared';
+import * as plugins from '../../plugins';
 
 const { common } = Kebab.factory;
 
-export const OverviewDetailsResourcesTab: React.SFC<OverviewDetailsResourcesTabProps> = ({
-  item: { buildConfigs, routes, services, pods, obj },
-}) => (
-  <div className="overview__sidebar-pane-body">
-    <PodsOverview pods={pods} obj={obj} />
-    <BuildOverview buildConfigs={buildConfigs} />
-    <NetworkingOverview services={services} routes={routes} />
-  </div>
+const getResourceTabSectionComp = (t) => (props) => (
+  <AsyncComponent {...props} loader={t.properties.loader} />
 );
 
+const getPluginTabSectionResource = (item) => {
+  const sections = plugins.registry
+    .getOverviewTabSections()
+    .filter((section) => item[section.properties.key]);
+  return sections.map((section) => getResourceTabSectionComp(section));
+};
+
+export const OverviewDetailsResourcesTab: React.SFC<OverviewDetailsResourcesTabProps> = ({
+  item,
+}) => {
+  const { buildConfigs, routes, services, pods, obj } = item;
+  const pluginComponents = getPluginTabSectionResource(item);
+  return (
+    <div className="overview__sidebar-pane-body">
+      <PodsOverview pods={pods} obj={obj} />
+      <BuildOverview buildConfigs={buildConfigs} />
+      {pluginComponents.map((Component, index) => (
+        <Component key={index} item={item} />
+      ))}
```

Comment:
This appears to be constantly mounted and unmounted. I am unsure why - even when using a static key it still happens. I'll continuing investigating.

Reply:
Yes, Not sure what is happening here, looking into this

Reply:
Jira Issue: https://jira.coreos.com/browse/ODC-2122